Fix the “Performance FAQ” content error (#8815)
flutter/website: src/perf/faq.md

    `debugProfileBuildsEnabled` and always call
    `Timeline.startSync(...)/finish`.
  * If you use IntelliJ, a GUI view of this data is available.
    Select **Track widget rebuilds**,

>     and you'll visually see which widgets rebuild
>     visually in your IDE.

Reviewer: **suggestion:** While you're here, consider simplifying this copy.
```suggestion
    and your IDE displays which the widgets rebuild.
```
